services/apis/fence/fenceSequences.js

```diff
@@ -112,5 +113,60 @@ module.exports = {
   async deleteFile(guid) {
     const res = await fenceTasks.deleteFile(guid);
     fenceQuestions.assertStatusCode(res, 204);
-  }
+  },
+
+
+  /**
+   * Cleans up fence's DBs for links and service accounts
+   * Takes the google and users utils as params
+   * @returns {Promise<void>}
+   */
+  async suiteCleanup(google, users) {
+    // unlock the lockable google project
+    await google.unlockGoogleProject(fenceProps.googleProjectDynamic);
+
+    // google projects to 'clean up'
+    const googleProjects = [
+      fenceProps.googleProjectA,
```

Comment:
`suiteCleanup` already includes ProjectA and ProjectWithComputeServiceAcct right now, I just added ProjectDynamic - from what I understand it cleans up the DB, in case a test crashed and didn't clean up